gsc/_t-univ-4.scm

```diff
@@ -3386,20 +3389,55 @@
 (univ-define-prim "##bignum.adigit-bitwise-andc1!" #f
   (make-translated-operand-generator
    (lambda (ctx return arg1 arg2 arg3 arg4)
-     ;; TODO: implement
-     (return arg1))))
+     (^ (^assign (^array-index (^bignum-digits arg1)
+                               (^fixnum-unbox arg2))
+                 (^cast* 'bigdigit
+                         (^bitand
+                          (^parens
+                           (^bitand (^parens
+                                     (^bitnot
+                                      (^array-index (^bignum-digits arg1)
+                                                    (^fixnum-unbox arg2))))
+                                    (^array-index (^bignum-digits arg3)
+                                                  (^fixnum-unbox arg4))))
+                          (^int univ-mdigit-base-minus-1))))
```

Comment:
Here you're using that the mdigits and adigits are the same in the universal backend, and using `univ-mdigit-base-minus-1` in an `adigit` routine.   `univ-adigit-base-minus-1` might be a more appropriate variable name.

Reply:
The universal backend uses the same width for mdigits and adigits, so either is OK.

Reply:
You are probably right... however this is nitpicking when there's a mountain of other stuff to resolve...